scripts/drtprod

```diff
@@ -456,6 +456,142 @@ WantedBy=multi-user.target
 EOF"
         roachprod ssh workload-ua -- "sudo systemctl daemon-reload; sudo systemctl enable tpcc.service; sudo systemctl start tpcc.service"
         ;;
+      "ldr")
+        roachprod create drt-ldr1 \
+          --clouds="gce" \
+          --gce-zones="us-east1-c" \
+          --nodes="5" \
+          --gce-machine-type="n2-standard-8" \
+          --local-ssd="true"  \
+          --gce-local-ssd-count="2" \
```

Comment:
With write heavy workloads, as we'll want to run on this cluster, we've encountered strange I/O effects when running on local SSDs.  I've converted all of my private LDR clusters to use PD as a result, and the I/O characteristics have been much better.  If budgeting permits, I think we should use PD here too, so that we're not investigating HW issues unnecessarily.

Reply:
I think for cost reasons we should try to stay within the envelope of sustainable local ssd operations? We have a fairly moderate `--concurrency` setting which should hopefully shield us from the worst behaviour of a very write heavy intensive workload on top of local ssds. If this is still an issue in practice, we can consider changing to PD.